src/components/TitleHeader/TitleHeader.js

```diff
@@ -4,34 +4,42 @@ import React from 'react'
 
 import { JIcon } from 'components/base'
 
-import titleHeaderStyle from './titleHeader.m.scss'
+import style from './titleHeader.m.scss'
 
-type TitleHeaderHandler = () => void
+type TitleHeaderHandler = () => any
 
 type Props = {|
   +onBack: ?TitleHeaderHandler,
+  +children: React$Node,
   +title: string,
 |}
 
 export function TitleHeader({
   onBack: handleClick,
+  children,
   title,
 }: Props) {
   return (
-    <div className={titleHeaderStyle.core}>
-      <h1 className={titleHeaderStyle.title}>{title}</h1>
+    <div className={style.core}>
       {handleClick && (
         <div
           onClick={handleClick}
-          className={titleHeaderStyle.back}
+          className={style.back}
         >
           <JIcon name='arrow-back-use-fill' size='medium' color='blue' />
         </div>
       )}
+      <h1 className={style.title}>{title}</h1>
```

Comment:
When `children` not define this block steal has `margin-bottom`, is it work as designed?

Reply:
Yes, its `margin-bottom` doesn't affect `aside`, because container is row.